src/components/Icon/GrantRequestSection/GrantRequestSectionFocused.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+
+export interface GrantRequestSectionIconProps {
+  sectionNumber: number
+}
+
+export function getNumberIcon(sectionNumber: number, white?: boolean) {
```

Comment:
`color` as an argument instead of white. The default can be a color.

Reply:
This function can be a function component.
`<NumberIcon value={number} color={color} />`